src/ncp/ncp_base.cpp

```diff
@@ -358,9 +361,6 @@ NcpBase::NcpBase(Instance *aInstance)
     otSrpClientSetCallback(mInstance, HandleSrpClientCallback, this);
 #endif
 #endif // OPENTHREAD_MTD || OPENTHREAD_FTD
-#if OPENTHREAD_CONFIG_DIAG_ENABLE
-    otDiagSetOutputCallback(mInstance, &NcpBase::HandleDiagOutput_Jump, this);
-#endif
```

Comment:
Should we keep this?

I think in the case that `OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE ` is not enabled, the `otDiagSetOutputCallback()` would never be called?

Reply:
Agree, I think we should keep this.